### PR TITLE
Add health_check django command (#217)

### PR DESCRIPTION
**Pull request by toopy**
In this PR we try to resolve this issue https://github.com/KristianOellegaard/django-health-check/issues/217 which looks interesting in some environments.

### CODE REVIEW

File: health_check/management/commands/health_check.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class Command(CheckMixin, BaseCommand):
+    help = "Run health checks and exit 0 if everything went well."
+
+    def handle(self, *args, **options):
+        # perform all checks
+        errors = self.errors
+
+        # use stdout io module for testing or sys.stdout by default
+        stdout = options.get('stdout', sys.stdout)
```
Comment (codingjoe) [inline health_check/management/commands/health_check.py]: No need to get it explicitly from the options, Django already does that for you. `sys.stdout` or whatever you pass via `call_command` is accessible via, `self.stdout`.
```suggestion
```

File: health_check/management/commands/health_check.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        for plugin in self.plugins:
+            style_func = self.style.SUCCESS if not plugin.errors else self.style.ERROR
+            stdout.write(
```
Comment (codingjoe) [inline health_check/management/commands/health_check.py]:
```suggestion
            self.stdout.write(
```

File: README.rst [hunk 3]
```diff
@@ -266,6 +266,26 @@ and customizing the ``template_name``, ``get``, ``render_to_response`` and ``ren
...
+--------------
+
+You can run django command `health_check` to perform command line checks in a shell,
+or periodically with a cron, as follow:
+
+.. code::
+
+    django-admin health_check
+
+Then you should get something like follow as output:
```
Comment (codingjoe) [inline README.rst]:
```suggestion
This should yield the following output:
```

File: README.rst [hunk 4]
```diff
@@ -266,6 +266,26 @@ and customizing the ``template_name``, ``get``, ``render_to_response`` and ``ren
...
+
+.. code::
+
+    DatabaseHealthCheck      ... working
+    CustomHealthCheck        ... unavailable: Something went wrong!
+
+In case of critical error (as with http version) the exit status will be `1`.
```
Comment (codingjoe) [inline README.rst]:
```suggestion
Similar to the http version, a critical error will case the command to quit with the exit code `1`.
```

File: README.rst [hunk 5]
```diff
@@ -266,6 +266,26 @@ and customizing the ``template_name``, ``get``, ``render_to_response`` and ``ren
         url(r'^ht/$', views.HealthCheckCustomView.as_view(), name='health_check_custom'),
     ]
 
+Django command
+--------------
+
+You can run django command `health_check` to perform command line checks in a shell,
```
Comment (codingjoe) [inline README.rst]:
```suggestion
You can run the Django command `health_check` to perform your health checks via the command line,
```

**Review by codingjoe — CHANGES_REQUESTED** [review]
Thank you so much @toopy for your contribution. Structured code, tests and well written documentation. If only all pull-requests where this carefully crafted 🥇 
I had some minor comments, once resolved I am happy to release this right away.

**Comment by toopy** [comment]
It's always a pleasure to contribute on a neat project ;) One more reason why I should use it. LGTM ?

**Review by codingjoe — APPROVED** [review]
👍 LGTM 👍

**Comment by codingjoe** [comment]
Released in [3.12.0](https://github.com/KristianOellegaard/django-health-check/releases/tag/3.12.0) 🎉

**Pull request merged**